tests/spread/integration/adduser/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Integration tests for adduser 
+
+execute: |
+  rootfs="$(install-slices adduser_bins)"
+  mkdir -p "$rootfs/{bin,lib,lib64}"
+  cp /bin/bash "$rootfs/bin"
+  chroot "$rootfs"
+  
+  test -f /etc/passwd
+  adduser --quiet --system testuser
```

Comment:
Why is this working? Isn't adduser a perl script? I don't see perl being included/chiselled anywhere

Reply:
You are right. This was working before because adduser was running outside of the chroot env. We need perl slice first before this test passes.